### ISSUE TITLE
False positive in PrintfScanfArgumentsInspector

### ISSUE THREAD

**Issue by Fonata** [kalessil/phpinspectionsea#1633]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         |  Php Inspections (EA Ultimate) 3.0.10|
| Language level | e.g. PHP 7.4                    |

Current behaviour
--
The following code gets flagged as "[EA] Pattern seems to be not valid."
````php
sprintf('%04.d', $my_integer);
````

Expected behaviour
--
This is a valid pattern producing leading zeros.

Environment details
---
IntelliJ IDEA 2020.2.2 (Ultimate Edition)
Build #IU-202.7319.50, built on September 15, 2020
Subscription is active until February 2, 2021
Runtime version: 11.0.8+10-b944.31 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
GC: ParNew, ConcurrentMarkSweep
Memory: 3014M
Cores: 4
Registry: go.run.processes.with.pty=false
Non-Bundled Plugins: com.intellij.apacheConfig, CFML Support, com.jetbrains.plugins.ini4idea, de.thomasrosenau.diffplugin, main.kotlin.com.jimschubert.intellij.swaggercodegen, name.kropp.intellij.makefile, net.seesharpsoft.intellij.plugins.csv, org.jetbrains.plugins.localization, org.jetbrains.plugins.rest, org.zalando.intellij.swagger, com.jetbrains.php, NEON support, com.kalessil.phpStorm.phpInspectionsUltimate, de.espend.idea.php.phpunit, org.jetbrains.plugins.phpstorm-remote-interpreter, com.jetbrains.php.behat, de.espend.idea.php.behat, com.jetbrains.twig, de.espend.idea.php.toolbox, de.espend.idea.php.annotation, fr.adrienbrault.idea.symfony2plugin, de.espend.idea.php.drupal, com.jetbrains.lang.ejs, org.jetbrains.plugins.vue, tv.twelvetone.intellij.plugins.intellivue, org.jetbrains.kotlin, lv.midiana.misc.phpstorm-plugins.deep-keys, com.jetbrains.php.drupal, Pythonid, ru.meanmail.plugin.pyannotations, org.jetbrains.plugins.go, net.ashald.envfile, ru.adelf.idea.dotenv

**Comment by ea-inspections-team** [comment]
Thank you for reporting @Fonata, the bung has been fixed.
